### DISCUSSION TITLE
Online study resources

### DISCUSSION THREAD

**Discussion by giscus** [gxanshu/gxanshu-site#100] [Q&A]
# Online study resources

List of all the free resources i have to share. these are illigal stuff so study at your own risk.

https://gxanshu.in/notes/online-study-resources/

**Comment by gxanshu** [comment]
Mega have 2GB download limit. but who needs to download this stuff anyways 😁. you can just add [mega chrome extension](https://chromewebstore.google.com/detail/mega/bigefpfhnfcobdlfbedofhhaibnlghod) to chrome based browsers and play unlimited videos without download.
